packages/cypress/test/components/cypress.json

```diff
@@ -0,0 +1,8 @@
+{
+  "component": {
+    "componentFolder": ".",
```

Comment:
I think this needs to be bazelfied... Not sure if this is the root cause.

Reply:
Yeah, we currently use a generated cypress plugin to add stuff to the config for test files. We will just need to do something similar for the paths of components / component tests